src/main/java/service/CollectionsMain.java

```diff
@@ -0,0 +1,30 @@
+package service;
+
+import model.collection.list.CustomArrayList;
+import model.collection.set.CustomHashSet;
+import model.ticket.Ticket;
+
+public class CollectionsMain {
+    public static void main(String[] args) {
```

Comment:
main method can be just 1 into the app. make this main method not main and call it from other main method

Reply:
Moved all code to the main method in TicketService